src/Test.CoreLib/src/Test.CoreLib.csproj

```diff
@@ -0,0 +1,252 @@
+      <Link>System\EETypePtr.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\Runtime\ExceptionIDs.cs">
+      <Link>System\Runtime\ExceptionIDs.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\Runtime\InteropServices\CallingConvention.cs">
+      <Link>System\Runtime\InteropServices\CallingConvention.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\Runtime\InteropServices\CharSet.cs">
+      <Link>System\Runtime\InteropServices\CharSet.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\Runtime\InteropServices\DllImportAttribute.cs">
+      <Link>System\Runtime\InteropServices\DllImportAttribute.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\Runtime\InteropServices\FieldOffsetAttribute.cs">
+      <Link>System\Runtime\InteropServices\FieldOffsetAttribute.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\Runtime\InteropServices\GCHandleType.cs">
+      <Link>System\Runtime\InteropServices\GCHandleType.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\Runtime\InteropServices\LayoutKind.cs">
+      <Link>System\Runtime\InteropServices\LayoutKind.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\Runtime\InteropServices\McgIntrinsicsAttribute.cs">
+      <Link>System\Runtime\InteropServices\McgIntrinsicsAttribute.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\Runtime\InteropServices\NativeCallableAttribute.cs">
+      <Link>System\Runtime\InteropServices\NativeCallableAttribute.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\Runtime\InteropServices\OutAttribute.cs">
+      <Link>System\Runtime\InteropServices\OutAttribute.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\Runtime\InteropServices\StructLayoutAttribute.cs">
+      <Link>System\Runtime\InteropServices\StructLayoutAttribute.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\Runtime\RuntimeExportAttribute.cs">
+      <Link>System\Runtime\RuntimeExportAttribute.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\Runtime\RuntimeImportAttribute.cs">
+      <Link>System\Runtime\RuntimeImportAttribute.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\String.cs">
+      <Link>System\String.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\ThrowHelpers.cs">
+      <Link>System\ThrowHelpers.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Runtime.Base\src\System\Void.cs">
+      <Link>System\Void.cs</Link>
+    </Compile>
+    <Compile Include="..\..\System.Private.CoreLib\src\System\Runtime\CompilerServices\EagerStaticClassConstructionAttribute.cs">
+      <Link>Runtime.Base\src\System\Runtime\CompilerServices\EagerStaticClassConstructionAttribute.cs</Link>
```

Comment:
Right. And it is harmless ... things can be marked with it even when `INPLACE_RUNTIME` is not defined.